### PR TITLE
[SD-2749] - Rewritten story cannot have new takes

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: server/features/content_publish.feature [hunk 1]
```diff
@@ -1632,4 +1632,101 @@ Feature: Content Publishing
...
+          "state": "draft",
+          "slugline": "Take-1 slugline",
+          "urgency": "4",
+          "pubstatus": "usable",
+          "subject":[{"qcode": "17004000", "name": "Statistics"}],
+          "anpa_category": [{"qcode": "A", "name": "Sport"}],
+          "anpa_take_key": "Take"
+      }]
+      """
+      And we post to "/archive/123/move"
```
Comment (sivakuna-aap) [inline server/features/content_publish.feature]: We can avoid this extra step as creating a item in a desk is a feature of Superdesk now.

Comment (mdhaman) [inline server/features/content_publish.feature]: still a valid scenario though.

Comment (sivakuna-aap) [inline server/features/content_publish.feature]: Agreed

File: server/apps/archive/archive_link.py [hunk 2]
```diff
@@ -59,3 +57,16 @@ def create(self, docs, **kwargs):
         doc.update(linked_item)
         build_custom_hateoas(CUSTOM_HATEOAS, doc)
         return [linked_item['_id']]
+
+    def _validate_link(self, target, target_id):
+        """
+        Validates the article to be linked
+        :param target: article to be linked
+        :param target_id: id of the article to be linked
+        :raises: SuperdeskApiError
```
Comment (sivakuna-aap) [inline server/apps/archive/archive_link.py]: Can we elaborate this doc bit more? 

`raises` should list all the possible exceptions the method is going to throw

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**